.test-infra/jenkins/job_LoadTests_Combine_Python.groovy

```diff
@@ -93,15 +93,23 @@ def loadTestConfigurations = { datasetName ->
         top_count            : 20,
       ]
     ],
-  ].each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+  ]
+  .each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+  .each{ test -> (jobType!='streaming') ?: addStreamingOptions(test) }
```

Comment:
Could you add a space before and after "!=" operator? That should make it look a little bit prettier.

Reply:
Sure, thanks :)